operator-common/src/main/java/io/strimzi/operator/cluster/model/Ca.java

```diff
@@ -67,6 +67,28 @@
 @SuppressWarnings("checkstyle:CyclomaticComplexity")
 public abstract class Ca {
 
+    public enum SecretEntry {
+        CRT(".crt"),
+        KEY(".key"),
+        P12_KEYSTORE(".p12"),
+        P12_KEYSTORE_PASSWORD(".password");
+
+        final String suffix;
+
+        private SecretEntry(String suffix) {
+            this.suffix = suffix;
+        }
+
+        public static String getDataForPod(Secret secret, String podName, SecretEntry entry) {
+            return secret.getData().get(getNameForPod(podName, entry));
+        }
+
+        public static String getNameForPod(String podName, SecretEntry entry) {
+            return podName + entry.suffix;
+        }
+
+    }
+
```

Comment:
please add Javadoc on the enum declaration, to all the enum values and public methods. We are going to have strict checks when moving to Java 17 so it's better to address missing doc right now. Thanks!